pkg/client/interfaces.go

```diff
@@ -78,19 +78,24 @@ type PreflightConfig struct {
 	Namespace string
 }
 
+// SonobuoyKubeAPIClient is the interface Sonobuoy uses to communicate with a kube-apiserver.
+type SonobuoyKubeAPIClient interface {
```

Comment:
Really?

Reply:
Better than having the sonobuoy client require a struct imo

but yes, the naming needs work